client/src/Components/Navbar.js

```diff
@@ -45,7 +46,7 @@ const Navbar = (props) => {
     history.push(route);
     if (route === '/signup') {
       dispatch({ type: LOGOUT });
-      fetch('/users/logout', { method: 'POST', credentials: 'same-origin' });
+      axios.post('http://localhost:3000/users/logout')
```

Comment:
We should not be writing localhost:3000 however - it should be /users/logout actually 

I do think we should make this an axios call though, and should be where our other API calls are being made